### PR TITLE
[Pornditos] get better image from scrape

### PR DESCRIPTION
**Pull request by stg-annon**

### CODE REVIEW

**Comment by bnkai** [comment]
It seems that all `cuchi mami` (`http://pornditos.com/t1/websites/cuchimami`) scenes use the x-images4 for the image shoots
Switching to something like the below seems to work
```yaml
              - regex: 'http://sm\.members\.khcdn\.com/shoots/([^/]+)/([^/]+)/rollover.*'
                with: "http://x-images5.bangbros.com/$1/shoots/$2/big_trailer.jpg"
              - regex: '5\.bangbros\.com/cuchimami'
                with: "4.bangbros.com/cuchimami"
```

**Comment by stg-annon** [comment]
something to note the slugs from this site map to `http://en.culioneros.com/videos/` which includes tags might want to use those links instead

File: scrapers/Pornditos.yml [hunk 1]
```diff
@@ -19,13 +24,44 @@ xPathScrapers:
...
+        postProcess:
+          - replace:
+              - regex: \n
+                with: " "
+      Performers:
+        Name: $info//a[contains(@href, "/models/")]/text()
+      Studio:
+        Name: $info//a[contains(@href, "/latest-videos/")]/text()
+      Image:
+        selector: //img[contains(@src, "/shoots/")][1]/@src # unsure why index 0 is not working
```
Comment (bnkai) [inline scrapers/Pornditos.yml:55]: Per w3c spec the first child of a node is accessed by 1 not 0 (<https://www.w3schools.com/xml/xpath_syntax.asp>)
comment can be removed

File: scrapers/Pornditos.yml [hunk 2]
```diff
@@ -19,13 +24,44 @@ xPathScrapers:
...
+        postProcess:
+          - replace:
+              - regex: '(http://x-images\d\.bangbros\.com/[^/]+/shoots/[^/]+).*'
+                with: "$1/big_trailer.jpg"
+      Date:
+        selector: $info//b[@class="t-font"]/text()
```
Comment (bnkai) [inline scrapers/Pornditos.yml:61]: The selector matches an extra text (the comma that separates the performers)
A more explicit selector should work fine
```yaml
        selector: $info//b[@class="t-font"][not(a)]/text()
```

File: scrapers/Pornditos.yml [hunk 3]
```diff
@@ -19,13 +24,44 @@ xPathScrapers:
...
+    common:
+      $info: //h2[@class="h2-trailer-block"]
+    scene:
+      Title: $info//span[@class="h2-trailer-style"]/text()
+      Details: 
+        selector: //p[@class="para-class t-para"]/text() 
+        postProcess:
+          - replace:
+              - regex: \n
+                with: " "
```
Comment (bnkai) [inline scrapers/Pornditos.yml:49]: I tried a few scenes without the pp and it seemed fine.
Do you have an example(url) of a scene that needs the pp ? (`\n` should have been removed by stash anyway)

File: scrapers/Pornditos.yml (hunk 1 above)
Comment (stg-annon) [reply scrapers/Pornditos.yml:55]: ahh wonderful the rebellious base 1

File: scrapers/Pornditos.yml (hunk 3 above)
Comment (stg-annon) [reply scrapers/Pornditos.yml:49]: yeah, I was having trouble getting it to recognize the line breaks from this link specifically
`http://en.culioneros.com/videos/carol-finds-a-guy-on-the-street-to-fuck-in-public_102934.htm`

**Pull request merged**